support-frontend/app/wiring/Controllers.scala

```diff
@@ -28,10 +26,9 @@ trait Controllers {
     appConfig.amazonPayConfigProvider,
     appConfig.recaptchaConfigProvider,
     paymentAPIService,
-    membersDataService,
+    appConfig.membersDataServiceApiUrl,
```

Comment:
we still need the URL, as we embed it in the contributions page, then the client side can call members data api for 
- sendGetExistingPaymentMethodsRequest
- fetchLatestOneOffContribution
- recurringContributionPaymentPlan